flood_adapt/adapter/sfincs_adapter.py

```diff
@@ -1081,7 +1080,13 @@ def _add_forcing_waterlevels(self, forcing: IWaterlevel):
             conversion = us.UnitfulLength(
                 value=1.0, units=self.settings.tide_gauge.units
             ).convert(us.UnitTypesLength.meters)
-            df_ts *= conversion
+
+            datum_height = self.settings.water_level.get_datum(
+                self.settings.tide_gauge.reference
+            ).total_height.convert(us.UnitTypesLength.meters)
+
+            df_ts *= conversion + datum_height
```

Comment:
These is a bug here! This should be done separately, since these are two different operation, while now you are multiplying with the sum.
```suggestion
            df_ts = conversion * df_ts + datum_height
```

Reply:
Oh that is interesting, good to know!
Nice find!